packages/lab/src/stepped-tracker/SteppedTracker.Provider.tsx

```diff
@@ -0,0 +1,30 @@
+import { type ReactNode, createContext, useContext } from "react";
+
+import type { Step } from "./Step";
+import type { SteppedTracker } from "./SteppedTracker";
+
+export const DepthContext = createContext<Step.Depth>(-1);
```

Comment:
I kind of think that if you group common components and export from the same file, then your getting the same thing.. So you could argue we have an equivalent solution that does not standout as using older conventions?

Remembering our initial chats, you liked this approach, particularly how the editor behaved. 
Problem is that we have not used it elsewhere so it might divide opinion from users?

Any Salt users have a view on this, do you do this yourselves ?

Reply:
removed namespaces, changed them to interfaces and types